solutions/src/main/java/ru/sberbank/school/task02/ExternalQuotesServiceC.java

```diff
@@ -0,0 +1,37 @@
+package ru.sberbank.school.task02;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+import lombok.Builder;
+import ru.sberbank.school.task02.exception.WrongSymbolException;
+import ru.sberbank.school.task02.util.Quote;
+import ru.sberbank.school.task02.util.Symbol;
+import ru.sberbank.school.task02.util.Volume;
+
+import static java.util.Arrays.asList;
+
+public class ExternalQuotesServiceC implements ExternalQuotesService {
+
+    public List<Quote> getQuotes(Symbol symbol) {
+        if (symbol.isCross()) {
+            throw new WrongSymbolException("Cross symbols detected");
+        }
+
+        List<Quote> quotes = asList(
+                buildQuote(symbol, 100, 10),
+                buildQuote(symbol, 500, 8),
+                buildQuote(symbol, 1000, 6),
+                buildQuote(symbol, 10_000, 5),
+                buildQuote(symbol, 100_000, 4),
+                buildQuote(symbol, 1_000_000, 6),
+                buildQuote(symbol, -1, 8)
+        );
+        return quotes;
+    }
+
+    private Quote buildQuote(Symbol symbol, int volume, int spread) {
+        return new Quote(symbol, Volume.from(volume), (BigDecimal.valueOf(80 - spread / 2)), (BigDecimal.valueOf(80 + spread / 2)));
```

Comment:
Line is longer than 120 characters (found 132).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-123789965>*